frontend/common/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import DialogComponent from "./Dialog";
+
+import Button from "../Button";
+
+export default {
+  component: DialogComponent,
+  title: "Components/Dialog",
+  args: {
+    title: "Dialog",
+  },
+  argTypes: {
+    title: {
+      name: "title",
+      type: { name: "string", required: true },
+      control: {
+        type: "text",
+      },
+    },
+    subtitle: {
+      name: "subtitle",
+      type: { name: "string", required: false },
+      control: {
+        type: "text",
+      },
+    },
+    content: {
+      name: "content",
+      type: { name: "string", required: false },
+      control: {
+        type: "text",
+      },
+    },
+    confirmation: {
+      name: "confirmation",
+      type: { name: "boolean", required: false },
+    },
+    color: {
+      options: ["primary", "secondary", "tertiary"],
+    },
+  },
+} as Meta;
+
+const TemplateDialog: Story = (args) => {
```

Comment:
The template starts off with the dialog hidden.  This makes it interactive but unfortunately it is hidden from the Chromatic snapshots.  Can we have it start with the dialog open?

Reply:
For example: 
![image](https://user-images.githubusercontent.com/8978655/165601482-cfbdabb6-0dcf-40c5-b4c7-e3d0b73f03b0.png)

Reply:
Resolved with [make dialog open by default in story](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2561/commits/f26b1b1b1d63f9554ff5dc98b650830ced33413f)